OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -63,6 +63,10 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Arc in WAngles, two values indicate variable arc.")]
 		public readonly WAngle[] LaunchAngle = { WAngle.Zero };
 
+		[Desc("Altitude where this bullet should explode when reached.",
+			"Negative values allow this bullet to pass cliffs and terrain bumps.")]
+		public readonly WDist ExplodeUnderThisAltitude = WDist.Zero;
```

Comment:
`AirburstAltitude` works for me.  The desc (which could also be added to Missile) explains how this can also be used as a workaround for mods that haven't configured their firing offsets yet.